spec/system/casa_cases/show_spec.rb

```diff
@@ -66,7 +66,7 @@
 
     it "can see next court date", js: true do
       expect(page).to have_content(
-        "Next Court Date: #{I18n.l(future_court_date.date, format: :day_and_date)}"
+        "Next Court Date: #{I18n.l(future_court_date.date, format: "%A, %-d-%^b-%Y")}"
```

Comment:
Not related but this is failing in master because the spec translation is blank padding the day but the translation in the view is not 🤔

Reply:
Oh no! Uh weird
Can you open a PR to fix it?